task-sdk/src/airflow/sdk/definitions/asset/__init__.py

```diff
@@ -319,58 +319,53 @@ class Asset(os.PathLike, BaseAsset):
         default=attrs.Factory(operator.attrgetter("asset_type"), takes_self=True),
         validator=[_validate_identifier],
     )
-    extra: dict[str, Any] = attrs.field(
```

Comment:
We can't remove this either.

Removing this parameter will result in a breaking change to any DAG which uses this -- that's not allowed.